### DISCUSSION TITLE
Configuration Errors

### DISCUSSION THREAD

**Discussion by APT256** [gravitational/teleport#10105] [Q&A]
Hello, I just installed Teleport 8.1 on a Raspberry Pi 3b running the latest Raspbian OS. There were no errors during the install, but when I try to generate the configuration file with this command:

sudo teleport configure --acme --acme-email=xxxxx@gmail.com --cluster-name=tele.xxx.xxx -o file

I get these errors:

/usr/local/bin/teleport: 1: S@���@8: not found
/usr/local/bin/teleport: 1: ELF�0: not found
/usr/local/bin/teleport: 1: Syntax error: Unterminated quoted string

Any ideas on what I need to do to correct these?

Thanks,
Jon

**Comment by zen** [comment]
How did you actually install the teleport?

**Reply by APT256** [reply]
I followed the steps from the installation guide:
curl -O https://get.gravitational.com/teleport-v8.1.3-linux-arm64-bin.tar.gz
tar -xzf teleport-v8.1.3-linux-arm64-bin.tar.gz
cd teleport
sudo ./install

**Comment by webvictim** [comment]
IIRC Raspberry Pi 3 uses a 32-bit chipset rather than 64-bit. You'll want the `arm` rather than `arm64` binary https://get.gravitational.com/teleport-v8.1.3-linux-arm-bin.tar.gz

**Comment by APT256** [comment]
Thanks for reminding me! The 64 bit OS is still in beta. For some reason I
thought it had been released already.

Jon

On Fri, Feb 4, 2022 at 2:28 PM Gus Luxton ***@***.***> wrote:

> IIRC Raspberry Pi 3 uses a 32-bit chipset rather than 64-bit. You'll want
> the arm rather than arm64 binary
> https://get.gravitational.com/teleport-v8.1.3-linux-arm-bin.tar.gz
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gravitational/teleport/discussions/10105#discussioncomment-2114029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACA6WJP2VJXSF4UK53AVVDTUZQZGZANCNFSM5NMO5WCQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Sent from Gmail Mobile
